.ado/jobs/cli-init.yml

```
@@ -165,7 +165,7 @@ parameters:
           - Name: ARM64ReleaseCsHermes
```

nit: the name of the matrix entry is lowercase by convention (even though the msbuild value is upper case)
```suggestion
          - Name: Arm64ReleaseCsHermes
```